src/test/java/codeu/model/data/HashtagTest.java

```diff
@@ -0,0 +1,69 @@
+package codeu.model.data;
+
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.UUID;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class HashtagTest {
+
+  @Test
+  public void testCreate() {
+    UUID id = UUID.randomUUID();
+    String content = "Football";
+    Instant creation = Instant.now();
+
+    Hashtag hashtag =
+        new Hashtag(id, content, creation, new ArrayList<String>(), new ArrayList<String>());
+
+    Assert.assertEquals(id, hashtag.getId());
+    Assert.assertEquals(content.toLowerCase(), hashtag.getContent());
+    Assert.assertEquals(creation, hashtag.getCreationTime());
+  }
+
+  @Test
+  public void testAddUser() {
+
+    String content = "Football";
+    Hashtag hashtag =
```

Comment:
You can use the `TestHashtagBuilder` here if you want.

Reply:
Will implement in the next PR